gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
```

Comment:
@matthewd @blomquisg  +1 for boring and descriptive, since it also covers multitenancy I just went with handled_list rather than expand_paginated_multitenancy_list.